src/dispatch/incident/models.py

```diff
@@ -300,6 +300,7 @@ class IncidentReadMinimal(IncidentBase):
     reporters_location: Optional[str]
     stable_at: Optional[datetime] = None
     tags: Optional[List[TagRead]] = []
+    tasks: Optional[List[TaskRead]] = []
```

Comment:
We should define a new `TaskReadMinimal` Pydantic model with the minimum fields we need and use that instead. Otherwise, loading all the fields in `TaskRead` will degrade performance.

Reply:
Done.